src/store-modules/search-store.js

```diff
@@ -305,9 +307,9 @@ const mutations = {
     }
     mediaToSet = mediaToSet.map((item) => decodeMediaData(item))
     _state[mediaPlural] = mediaToSet
-    _state.pageCount = pageCount
-    _state.count[mediaPlural] = mediaCount || 0
+    _state[`${mediaPlural}Count`] = mediaCount || 0
     _state[`${mediaType}Page`] = page || 1
+    _state.pageCount[mediaPlural] = pageCount
```

Comment:
```suggestion
    _state.pageCount[mediaPlural] = pageCount || 0
```

Reply:
Thanks! I commented some tests that are throwing errors because of this `pageCount` object. This is obviously undesirable, we need those tests so that name may change.